code/game/objects/items/contraband.dm

```diff
@@ -3,12 +3,13 @@
 //Illicit drugs~
 /obj/item/storage/pill_bottle/happy
 	name = "Happy pills"
-	desc = "Highly illegal drug. When you want to see the rainbow."
+	desc = "Real fun drugs, for when you want to see the rainbow. Happy happy joy joy!"
```

Comment:
```suggestion
	desc = "Real fun drugs for when you want to see the rainbow. Happy happy joy joy!"
```
this is pretty nitpicky, but I don't think this comma is neccessary

Reply:
I think the comma breaks up the description of them, then it describes what they actually do

Reply:
Saying "for when.." also breaks them up IMO. It's not like either way is wrong, I just think removing the comma makes it flow better.